tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -1510,14 +1510,10 @@ namespace tuplex {
             if(!baseURI.isLocal())
                 throw std::runtime_error("VFS not supporting mkdir yet!");
 
-            // check if folder exists, if so delete its contents (overwrite mode!)
-            // then, put part files in it!
+            // check if folder exists, if it does not, create it
             auto vfs = VirtualFileSystem::fromURI(baseURI);
-            if(baseURI.exists())
-                vfs.remove(baseURI);
-
```

Comment:
1. file -> okay
2. dir -> must not exist or be empty
3. if non-empty error

Reply:
I think we probably should have some helper function validateOutputSpecification(...) or so, which we call before actually planning anything to not have this odd behavior of executing something first then realizing the output can't be properly written. This comes at the cost that during job execution some change to a file-system might have occurred. I.e., when actually writing we need to be careful on this part. Yet, I feel it should be user responsibility to ensure that the configuration doesn't change while executing a job.

Reply:
other option would be to call the validateOutputSpecification twice, yet this could be costly.

Reply:
done.